packages/common/infra/src/blocksuite/initialzation/index.ts

```diff
@@ -0,0 +1,289 @@
+import { assertExists } from '@blocksuite/global/utils';
+import type { Page, PageMeta, Workspace } from '@blocksuite/store';
+import type { Schema } from '@blocksuite/store';
+import type { createStore, WritableAtom } from 'jotai/vanilla';
+import { nanoid } from 'nanoid';
+
+export async function initEmptyPage(page: Page, title?: string) {
+  await page.waitForLoaded();
+  const pageBlockId = page.addBlock('affine:page', {
```

Comment:
blocksuite has a new way to initial block structure

https://block-suite.com/data-persistence.html#block-tree-initialization-basics

```ts
await page.load(() => {
  const pageBlockId = page.addBlock('affine:page', {
    title: new page.Text(title ?? ''),
  });
  page.addBlock('affine:surface', {}, pageBlockId);
  const noteBlockId = page.addBlock('affine:note', {}, pageBlockId);
  page.addBlock('affine:paragraph', {}, noteBlockId);
});
```

Reply:
Those codes I just moved index.ts, maybe refactor next time, I can add some comments.